http/http_server.hpp

```diff
@@ -19,7 +19,9 @@
 #include <memory>
 #include <utility>
 #include <vector>
-
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
```

Comment:
I wonder if we should get rid of the ifdef here.. upstream bmcweb has been doing that